app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+  def home
```

Comment:
def home; end or skip at all, up to you
As for me good solution when you just left a comment that there is such method
# home
and skip initialization